conda.recipe/build.sh

```diff
@@ -3,3 +3,9 @@
 python setup.py install --single-version-externally-managed --record=record.txt
 
 cp bdist_conda.py ${PREFIX}/lib/python${PY_VER}/distutils/command
+
+# required for distutils included in setuptools >=60.0.0
+# see also https://setuptools.pypa.io/en/latest/history.html#v60-0-0
+SETUPTOOLS_DISTUTILS="${PREFIX}/lib/python${PY_VER}/site-packages/setuptools/_distutils/command/"
+mkdir -p "${SETUPTOOLS_DISTUTILS}"
+cp bdist_conda.py "${SETUPTOOLS_DISTUTILS}"
```

Comment:
Forgive my lack of knowledge, will this be run during installation? There is no other way to add additional bdist commands to setuptools?

Reply:
yeah... those lines also look wired ... I did not find any distutils plugin mechanism (like pytest has) except the solution chosen in the past where something is placed into the distutils command folder ... 

Seems like we doing this since while, see line 5 in that file https://github.com/conda/conda-build/pull/4368/files#diff-2f7714f3308cb222f2b5b3bba3d7395b040ef39f5303b5917175630860f8099eR5

`cp bdist_conda.py ${PREFIX}/lib/python${PY_VER}/distutils/command`

All we do with this PR is to also copy the same file into the distutils folder that got vendored by setuptools (once setuptools are imported you can not import distutils from python anymore.)

Reply:
refs:
* https://docs.python.org/3/distutils/extending.html
* https://setuptools.pypa.io/en/latest/userguide/extension.html

Maybe an setup.py entry similar to this would work:
```
setup(
    # ...
    entry_points={
        "distutils.commands": [
            "bdist_conda = conda_build.bdist_conda:bdist_conda",
        ],
    },
)
```
Requires to move `bdist_conda.py` to somewhere in `./conda_build/...` which should be possible.

It also requires, that the bdist_conda class in conda_build.bdist_conda is a setuptools.Command subclass, which is the case.

Reply:
If it works, it would be a more future-proof approach than modifying the distutils package (under the setuptools package). So I'd be in favor of fixing the problem that way, but I also see it increasing the scope of your change a bit.

Reply:
agree, will give it a try.